static/styles/recent_topics.css

```diff
@@ -148,10 +162,31 @@
             background-color: hsl(105, 2%, 50%);
         }
 
+        .unread_count_pm {
+            /* 10px of unread count + 23px for bell icon */
+            margin-right: 33px;
+            /* match the opacity with topic unread count without hover */
+            opacity: 0.7;
+        }
+
         .unread_hidden {
             visibility: hidden;
         }
 
+        .user_circle {
+            /* size the user activity circle */
+            min-width: 7px;
+            height: 7px;
+            margin-left: 8px;
+            top: 0;
+        }
```

Comment:
How are these numbers picked? And how do they related to the styles in `user_circles.css`?

Reply:
@timabbott The size of the `user_circle` is kept constant with the user_circle in left sidebar. But it was looking weird so I decreased the size by 1px.